src/Row.php

```diff
@@ -173,4 +177,19 @@ public function setPreparationCallback(Closure $preparationCallback = null)
     {
         $this->preparationCallback = $preparationCallback;
     }
+
+    private function cleanValue($value)
+    {
+        if(!is_string($value)){
+
+            return $value;
+        }
+        $cleaned = preg_replace('~^[\s\x{FEFF}\x{200B}]+|[\s\x{FEFF}\x{200B}]+$~u', '', $value) ?? trim($value);
```

Comment:
Please add a comment what this preg_replace does. 
Also I think this should be `?:` instead of `??`